notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb

```diff
@@ -4,31 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>adding comments to explain the purpose of the code blocks/lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>